.travis.yml

```diff
@@ -40,11 +40,8 @@ install:
 - export PATH=$HOME/.local/bin:$PATH;
 
 # install openmdao and sphinx-travis
-- git clone https://github.com/openmdao/openmdao.git;
-- cd openmdao;
+- pip install `openmdao==3.1.0`
```

Comment:
I think the backticks are causing Travis to error out, don't think they're needed here.

Reply:
Do we need to specify 'openmdao==3.1.0' or can we instead leave it without a specific version so that when openmdao is updated in the future we do not need to change our yml file to have travis catch issues?